doc/cookbook/source/examples/regression/multiple_kernel_learning_regression.rst

```diff
@@ -0,0 +1,55 @@
+=======================================
+Multiple Kernel Learning for Regression
+=======================================
+
+Multiple kernel learning (MKL) is based on convex combinations of arbitrary kernels over potentially different domains.
+
+.. math::
+
+    {\bf k}(x_i,x_j)=\sum_{i=1}^{K} \beta_k {\bf k}_i(x_i, x_j)
+
+where :math:`\beta_k > 0`, :math:`\sum_{k=1}^{K} \beta_k = 1`, :math:`K` is the number of sub-kernels, :math:`\bf{k}` is a combined kernel, :math:`{\bf k}_i` is an individual kernel and :math:`{x_i}_i` are the training data.
+
+Regression is done by using :sgclass:`CSVMLight`.
```

Comment:
MKL regression needs an SVM solver as input. We here use SVMLight. 


Put this further down